### ISSUE TITLE
Group channel creation with user ids on React Native 0.61x not working as expected

### ISSUE THREAD

**Issue by hsb1007** [sendbird/SendBird-JavaScript#109]
The exact same code perfectly works fine on RN 0.59 but when I update it to 0.61, Group channel creation fails to include the other users.

E.g) 
If I include user myself, ben and grace,  channel object doesn't include 'ben' and 'grace'

Does anyone have the same issue?

I am assuming the code works fine because it worked fine on RN0.59

**Comment by AlexOrrSB** [comment]
I was unable to reproduce this using our React Native sample app and upgrading to React Native version 0.61.2 and the newest version of the SendBird SDK. Could you provide additional steps to reproduce (SDK version numbers, method calls, snippets, etc.). It would also be helpful if you could provide the HTTP request that is sent when you try this.

**Comment by hsb1007** [comment]
Thanks @AlexOrrSB for your help.

I am using below package

- React : 16.9.0
- React-native: 0.61.4 (tried with 0.61.5)
- sendbird:  3.0.114  (tried with 3.0.117)



This is how I connect to Sendbird..

![image](https://user-images.githubusercontent.com/4149765/74285274-9a1a0700-4d79-11ea-8190-23e62b8762c9.png)

And this is how I create a group channel where userList is an array of user ids in number 

![image](https://user-images.githubusercontent.com/4149765/74285296-a3a36f00-4d79-11ea-9a6a-b69bbdcb00f9.png)

![image](https://user-images.githubusercontent.com/4149765/74285304-a8682300-4d79-11ea-9200-5e9aa836a05c.png)




Apology the code insertion didn't work as expected so I had to put screenshots.

This exact code works perfectly on RN 0.59 but not on RN 0.61

Any insight would be appreciated!

**Comment by AlexOrrSB** [comment]
Thanks! This seemed to work fine for me using the versions you specified above... Could you share the app id, and possibly a ballpark timestamp of when you experienced this issue. 

I assume when you log user list before the request as in the example above that both userIds are in the list? Do both users exist in your SendBird application?

**Comment by hsb1007** [comment]
@AlexOrrSB  Yes! the app ID is "19DAE590-E41F-4652-B9FF-D0C763C78C5C" and it happened around  00:00 GMT 12th Feb 2020!

And yes I have made sure the users are already in the Sendbird application.


So when we create a channel, it got created without the rest of the users as if the users weren't invited at all in the first place.

And even when I pass different user ids, who are in the Sendbird application, the channel url in the response is also same..


I am still investigating but have not been lucky yet..

Appreciate your help! any advice will help us alot! 

Cheers

**Comment by hsb1007** [comment]
@AlexOrrSB any thought or advice at all?  Thanks Alex..

**Comment by AlexOrrSB** [comment]
@hsb1007 I'm double checking this
